README.md

```diff
@@ -102,4 +102,20 @@ Name=Do a thing
 
 ```
 
+# Flatpak build
+
+Installing by flatpak is also possible
+
+ - Clone this repo
+ - `flatpak-builder build .flatpak-manifest.yaml --user --install --force-clean`
+ - This will build and install kiot as a flatpak fetching all dependencies
+
+## Notes:
+
+The flatpak will not autostart.
+The config file now needs to be at ~/.var/app/org.davidedmundson.kiot/config/kiotrc
```

Comment:
with the arg `--filesystem=xdg-config/kiotrc`, this wouldn't be nescessary. Without a UI, I'd suggest to add this. With a UI, it can make sense to have this in it's own location. What do you think?

Reply:
smart